test/e2e/specs/editor/blocks/buttons.spec.js

```diff
@@ -30,7 +30,9 @@ test.describe( 'Buttons', () => {
 		editor,
 		page,
 	} ) => {
-		await editor.canvas.click( 'role=button[name="Add default block"i]' );
+		await editor.canvas
+			.locator( 'role=button[name="Add default block"i]' )
+			.click();
```

Comment:
Sidenote: This is a really common pattern; I wonder if we should extract it into a utility method.

Reply:
```suggestion
		await editor.canvas
			.getByRole( 'button', {
			    name: "Add default block"
			})
			.click();
```

Nit: I have previously been  advised to prefer `getByRole` wherever possible.

Reply:
It's probably not worth refactoring old code for this standard, especially if we plan to extract this in helper.

Both ways are fine, but using RTL-style locators is preferred.